spec/factories/usuario.rb

```diff
@@ -2,11 +2,11 @@
 
 FactoryGirl.define do
   factory :usuario do
-    nombre            { Faker::Name.first_name }
-    apellidos         { Faker::Name.last_name }
-    nick              { |u| "a" + Faker::Internet.user_name(u.nombre)[0..12]+SecureRandom.hex(3) }
+    nombre            { FFaker::Name.first_name }
+    apellidos         { FFaker::Name.last_name }
+    nick              { |u| "a" + FFaker::Internet.user_name(u.nombre)[0..12]+SecureRandom.hex(3) }
```

Comment:
Unnecessary spacing detected.<br>Surrounding space missing for operator '+'.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.